### DISCUSSION TITLE
Is this an issue with TShock or not?

### DISCUSSION THREAD

**Discussion by mk181** [Pryaxis/TShock#2542] [Q&A] [answered]
Whenever I try to start my server I get the error given in the crash log file below and can't find any fixes for it.
[client-crashlog (2).txt](https://github.com/Pryaxis/TShock/files/7740618/client-crashlog.2.txt)

**Comment by Arthri** [comment, marked as answer]
Can you show us your server config and flags you're passing into the server?

**Reply by mk181** [reply]
The server never generated a config

**Reply by Arthri** [reply]
According to the log, you're passing something into the server, e.g.: `terrariaserver.exe -world ... -port ...`

**Reply by mk181** [reply]
ah nvm, turns out I didn't give a value to the flags. now it starting up
